services/core/shared/core/compat/sdk_v5/paramUtils.js

```diff
@@ -19,7 +19,8 @@ const {
 
 const normalizeRangeParam = (params, property) => {
 	if (typeof params[property] === 'string' && params[property].includes(':')) {
-		const [from, to] = params[property].split(':');
+		const from = Number(params[property].split(':')[0]);
+		const to = Number(params[property].split(':')[1]);
```

Comment:
Kindly consider including the following check and a corresponding test case for the negative scenario.

```suggestion
		const [fromStr, toStr] = params[property].split(':');
		const from = Number(fromStr);
		const to = Number(toStr);
		if (Number.isNaN(from) || Number.isNaN(to)) throw new ValidationException(`Invalid (non-numeric) '${property}' range values supplied: ${params[property]}.`);
```